kepler/tasks.py

```diff
@@ -76,12 +78,33 @@ def submit_to_dspace(job, data):
         pkg.metadata = _fgdc_to_mods(get_fgdc(data))
         with tempfile.NamedTemporaryFile(suffix='.zip') as fp:
             pkg.write(fp)
-            handle = sword.submit(current_app.config['SWORD_SERVICE_URL'], fp.name,
-                                  auth=(username, password))
+            handle = sword.submit(current_app.config['SWORD_SERVICE_URL'],
+                                  fp.name, auth=(username, password))
         job.item.handle = handle
         db.session.commit()
 
 
+def get_geotiff_url_from_dspace(job):
+    """Retrieve the GeoTIFF URL from a DSpace Handle.
+
+        .. note:: assumes the OAI-ORE only contains a single TIFF.
+
+        :param job: :class:`~kepler.models.Job`
+    """
+    handle = job.item.handle.replace('http://hdl.handle.net/', '')
+    ore_url = current_app.config['OAI_ORE_URL'] + handle + '/ore.xml'
+    r = requests.get(ore_url)
+    r.raise_for_status()
+    doc = ET.fromstring(r.text)
+    image_urls = doc.findall(
+                './/{http://www.w3.org/1999/02/22-rdf-syntax-ns#}Description')
+    for url_element in image_urls:
+        url = url_element.get(
+                        '{http://www.w3.org/1999/02/22-rdf-syntax-ns#}about')
+        if '.tif?' in url:
+            job.item.geotiff_url = url
```

Comment:
I'm reasonably sure this won't work with a real db as I did nothing to add the `geotiff_url` field to an `Items` table... because I don't see where to do that and I wanted this to fail miserably and it didn't.